public/app/features/logs/components/log-context/LogRowContextModal.tsx

```diff
@@ -300,6 +321,28 @@ export const LogRowContextModal: React.FunctionComponent<LogRowContextModalProps
           Showing {context.before.length} lines {logsSortOrder === LogsSortOrder.Descending ? 'after' : 'before'} match.
         </div>
       </div>
+      {getRowContextQuery && (
```

Comment:
I have tried it with elastic and it shows the button (that does nothing when pressed) even for elasticsearch 
<img width="1496" alt="image" src="https://user-images.githubusercontent.com/30407135/233018637-67a487cc-08d2-4762-86fc-9860fa92c049.png">

Reply:
Fixed in [e1c416a](https://github.com/grafana/grafana/pull/66777/commits/e1c416ac548d9474e6d3f020e39764b6b6338f7e)